src/handlers/ens.ts

```diff
@@ -490,55 +484,14 @@ export const fetchAccountPrimary = async (accountAddress: string) => {
   };
 };
 
-export const fetchProfile = async (ensName: string) => {
-  const [
-    resolver,
-    records,
-    coinAddresses,
-    images,
-    owner,
-    { registrant, registration },
-    primary,
-  ] = await Promise.all([
-    fetchResolver(ensName),
-    fetchRecords(ensName),
-    fetchCoinAddresses(ensName),
-    fetchImages(ensName),
-    fetchOwner(ensName),
-    fetchRegistration(ensName),
-    fetchPrimary(ensName),
-  ]);
-
-  const resolverData = {
-    address: resolver?.address,
-    type: resolver?.address === ensPublicResolverAddress ? 'default' : 'custom',
-  };
-
-  return {
-    coinAddresses,
-    images,
-    owner,
-    primary,
-    records,
-    registrant,
-    registration,
-    resolver: resolverData,
-  };
-};
-
-export const fetchProfileRecords = async (ensName: string) => {
-  const [records, coinAddresses, images] = await Promise.all([
-    fetchRecords(ensName),
-    fetchCoinAddresses(ensName),
-    fetchImages(ensName),
-  ]);
-
-  return {
-    coinAddresses,
-    images,
-    records,
-  };
-};
+export function prefetchENSIntroData() {
+  for (const name of ensIntroMarqueeNames) {
+    prefetchENSAddress(name, { cacheFirst: true });
```

Comment:
`cacheFirst` means that it will use the data in local storage (if exists), otherwise it will perform network request